src/layout/Modal.js

```diff
@@ -25,10 +25,11 @@ let Modal = ({ isOpen, children, style = {} }) => (
           style={{
             backgroundColor: '#fff',
             borderRadius: 5,
-            maxWidth: 'fit-content',
+            maxWidth: 640,
+            overflowX: 'scroll',
             // minHeight: 300,
             margin: '0 auto',
-            padding: 30,
+            border: '30px solid white',
```

Comment:
@daedalus28 we could also make the component take an `innerStyle` prop for this div and pass this through from grey vest

Reply:
We should definitely do that. Also, the 30px border is super weird 🤔

Reply:
And if that's too tricky, we can just put a class on it and override in grey vest with css - but the 30px should not be in the core modal code